src/Controller/API/AcademicYears.php

```diff
@@ -27,17 +29,20 @@ public function getOne(
         Request $request,
         CourseRepository $courseRepository,
         SerializerInterface $serializer,
-        ApiResponseBuilder $builder
+        ApiResponseBuilder $builder,
+        AcademicYearFactory $academicYearFactory
     ): Response {
+        $year = (int) $id;
```

Comment:
I don't know if this will work, but I bet symphony will cast this for you if you declare it in the `getOne` above.

Reply:
this works! changed it once more.

Reply:
:heavy_check_mark: